.eslintrc

```diff
@@ -265,4 +265,13 @@
             },
         },
     ],
+    "ignorePatterns": [
+        "tests/files/broken-trampoline.js",
+        "tests/files/cycles/flow-typeof.js",
+        "tests/files/default-export-from.js",
+        "tests/files/flowtypes.js",
+        "tests/files/issue-195/Endpoints.js",
+        "tests/files/named-trampoline.js",
+        "tests/files/trampoline.js",
```

Comment:
fine for this PR, but what were the linting errors? if it's not impractical, i'd rather disable unimportant rules to ensure that files can still be linted for parse correctness

Reply:
here are some samples:

```sh
(node:34323) DeprecationWarning: "first" rule is using `context.getDeclaredVariables()`, which is deprecated and will be removed in ESLint v9. Please use `sourceCode.getDeclaredVariables()` instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

Reply:
those are just warnings, i don't think they'll fail the build?